client/src/actions/team.js

```diff
@@ -298,3 +319,75 @@ export const updateTeam = (teamId, teamData) => async (dispatch) => {
     dispatch(errorActionGlobalCreator(error));
   }
 };
+
+/**
+ * This action creator will be called when a user want to delete their account
+ *
+ * @param {*} teamId id of the team
+ * @param {*} teamData data object of the data to be deleted
+ * @returns
+ */
+export const resetTeamData = (teamId, isDelete) => async (dispatch) => {
+  try {
+    const body = {
+      isDeleteFlag: isDelete,
+    };
+    await api.resetTeamData(teamId, body);
+    dispatch({
+      type: DELETE_TEAM_PUBLICATIONS,
+    });
+    dispatch({
+      type: RESET_ACHIEVEMENT,
+    });
+    dispatch({
+      type: RESET_TEAM,
+    });
+
+    dispatch(successMessageCreator('Your account data has been reset.'));
+  } catch (error) {
+    dispatch(errorActionGlobalCreator(error));
+  }
+};
+
+/**
+ * This action creater find/create a new theme and update it in team data.
+ * @param {*} teamId
+ * @param {*} themeData
+ * @returns
```

Comment:
Please don’t add a docstring and leave fields empty. You should either populate them or remove the docs entirely

Reply:
This was from the merge, i get weird stuff like this on my laptop ig